casanovo/data/ms_io.py

```diff
@@ -2,17 +2,99 @@
 
 import collections
 import csv
+import logging
 import operator
 import os
 import re
+from datetime import datetime
 from pathlib import Path
-from typing import List
+from socket import gethostname
+from sys import argv
+from time import time
+from typing import List, Optional, Dict
 
 import natsort
+import numpy as np
+import torch
+from pandas import DataFrame
 
 from .. import __version__
 from ..config import Config
 
+SCORE_BINS = [0.0, 0.5, 0.9, 0.95, 0.99]
+
+logger = logging.getLogger("casanovo")
+
+
+def get_score_bins(
+    results_table: DataFrame, score_bins: List[float]
+) -> Dict[float, int]:
+    """
+    From a list of confidence scores, return a dictionary mapping each confidence score
```

Comment:
**todo:** According to the [docstring style guide](https://peps.python.org/pep-0257/), the first line should be a short, concise summary, followed by a blank line, then optionally providing more information in the following lines.